_data/events.yml

```diff
@@ -38,7 +38,8 @@
   url: https://local.hackcu.org
   
 - name: HackCU IV
-  location: TBD
+  date: 2018-02-24 8:00 AM
+  location: SEEC Building (East Campus)
   image-url: assets/img/hackcu.png
   line: Subscribe to be the first to know when applications go live.
```

Comment:
I would suggest that we wait until @casassg deploys the backend for public. And then update the description also in the same PR